content/en/docs/generating-objects/instantiate-methods.md

```diff
@@ -0,0 +1,644 @@
+---
+title: "Instantiate Methods"
+images: []
+menu:
+docs:
+parent: "generating-objects"
+identifier: "instantiate-methods"
+weight: 32
+---
+
+There are several ways of creating an object. For example, you can initialize an instance of an object using its constructor, or you can use the factory method it has.
```

Comment:
It is better to mention that each test can use a different way of instantiating.